lib/rubocop/cop/root_cops/no_index_on_add_column.rb

```diff
@@ -0,0 +1,31 @@
+module RuboCop
+  module Cop
+    module RootCops
+      class NoIndexOnAddColumn < Cop
+        ACTIVE_RECORD_REGEX = /ActiveRecord::Migration\[\d\.\d\]/.freeze
```

Comment:
Should this be `\d+` to handle higher point releases?

Reply:
Probably; this is pervasive in the linter so I'll follow up with a later PR.